webpack.config.ts

```diff
@@ -43,6 +38,13 @@ interface Configuration extends webpack.Configuration {
 
 const {env} = process;
 
+if (env.EXTRACT_IOS_DEVICE_NAMES) {
```

Comment:
I'm not sure it makes sense to keep this in webpack if it doesn't run automatically, might fit better as a npm script in package.json

Reply:
Makes sense. I'm removing from here. Thank you.